package.json

```diff
@@ -46,36 +46,36 @@
     "@babel/preset-typescript": "7.16.7",
     "@mui/material": "^5.9.1",
     "@mui/styled-engine-sc": "^5.8.0",
-    "@mui/styles": "^5.9.1",
-    "@nrwl/cli": "15.0.0",
-    "@nrwl/cypress": "15.0.0",
-    "@nrwl/eslint-plugin-nx": "15.0.0",
-    "@nrwl/express": "15.0.0",
-    "@nrwl/jest": "15.0.0",
-    "@nrwl/linter": "15.0.0",
-    "@nrwl/node": "15.0.0",
-    "@nrwl/react": "15.0.0",
-    "@nrwl/tao": "15.0.0",
-    "@nrwl/web": "15.0.0",
-    "@nrwl/workspace": "15.0.0",
+    "@nrwl/js": "16.0.3",
+    "@nrwl/tao": "16.0.3",
```

Comment:
Why was `@mui/styles` removed in 740e3d57d30afe8288dacbc00c961da58658b277? Also, `@nrwl/js` has been deprecated in favor of `@nx/js`, whereas `@nrwl/tao` has been deprecated in favor of `nx`.

Reply:
https://github.com/mui/material-ui/issues/32142#issuecomment-1090002923

That github post is related on the decision of removing `@mui/styles` and related to the `@nrwl` packages remaning, the NX migration didn't change them automatically and when I tried to do it manually the Nx version of these two doesn't seem to be in npm (even though Nx in their documentation they speak of nx/tao and nx/js)